static-site/src/components/ListResources/IndividualResource.tsx

```diff
@@ -43,6 +43,14 @@ export const ResourceLink = styled.a`
   &:hover {
     color: ${LINK_HOVER_COLOR} !important;
   }
+
+  &:not(:hover)::before {
+    content: "${(props) => props.$content}";
+  }
+
+  &:hover::before {
+    content: "${(props) => props.$hoverContent || props.$content}";
+  }
```

Comment:
_non-blocking_

This isn't what I meant… but that's on me for not specifying it and assuming you'd understand. I meant to include the content in the HTML itself (rather than using pseudo-elements in CSS) and only show the last label/name until hovered.

Don't worry about doing anything different now. I wanted to point out that I think there's a simplification here that's not mixing runtime inspection of CSS states with React component rendering.

Reply:
Oh gotcha, thanks for clarifying. I'll just revert bfbf4da116563cdc83913db5ff2295dc53ef16b8 and merge this PR. Maybe will try out that idea later...